src/components/IndividualArticle.jsx

```diff
@@ -1,31 +1,60 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getArticleById } from "../../apicalls";
+import { getArticleById, getCommentByArticleId } from "../../utils/apicalls";
+import CommentCard from "./CommentCard";
+import { dateTimeSeperator } from "../../utils/function";
 
 const IndividualArticle = () => {
   const [article, setArticle] = useState({});
   const { article_id } = useParams();
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
 
   useEffect(() => {
+    setIsLoading(true);
     getArticleById(article_id).then((article) => {
       setArticle(article);
     });
+    getCommentByArticleId(article_id).then((comments) => {
+      setComments(comments);
+      setIsLoading(false);
+    });
   }, []);
 
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
   return (
-    <div className="article-card">
-      <h2>{article.title}</h2>
-      <p>By: {article.author}</p>
-      <img src={article.article_img_url} alt={article.topic} />
-      <section>
-        <p>{article.body}</p>
-      </section>
-      <aside>
-        <p>Date Posted: {article.created_at}</p>
-        <p>Number of Comments: {article.comment_count}</p>
+    <>
+      <div className="individual-card">
+        <h2>{article.title}</h2>
+        <p>By: {article.author}</p>
+        <img src={article.article_img_url} alt={article.topic} />
+        <section>
+          <p>{article.body}</p>
+        </section>
+        <aside>
+          <p>Date Posted: {article.created_at}</p>
+          <p>Number of Comments: {article.comment_count}</p>
+        </aside>
+        <button>Upvote</button>
         <p>Votes: {article.votes}</p>
-      </aside>
-    </div>
+        <button>DownVote</button>
+      </div>
+      <h2>Comments</h2>
+      {comments.map(({ comment_id, body, author, votes, created_at }) => {
```

Comment:
If the comments array is empty (i.e. there are no comments), there is currently nothing to tell the user this. Could we add some code to account for this situation please?